bazel/BUILD.redis

```diff
@@ -48,13 +48,8 @@ genrule(
         "redis-cli",
     ],
     cmd = """
-    mkdir tmp-redis-bin
-    cp -rf $(locations :redis) ./tmp-redis-bin/
-    cp tmp-redis-bin/redis-server $(location redis-server)
-    chmod +x $(location redis-server)
-    cp tmp-redis-bin/redis-cli $(location redis-cli)
-    chmod +x $(location redis-cli)
-    rm -rf tmp-redis-bin
```

Comment:
is this change no longer necessary?

Reply:
This is in this PR https://github.com/ray-project/ray/pull/29179
Otherwise, I can't rebuild it locally...

Reply:
Let me merge to the upstream